src/tasks/Common/NotNullAttribute.cs

```diff
@@ -0,0 +1,7 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
It might be better to just include whole System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/NullableAttributes.cs